splunk_add_on_ucc_framework/install_python_libraries.py

```diff
@@ -55,6 +61,24 @@ def _subprocess_call(
         raise e
 
 
+def _subprocess_run(
```

Comment:
This looks very similar to `_subprocess_call`. I would suggest refactoring `_subprocess_call` into returning full response and using it everywhere.